### PR TITLE
Invitation layout

### PR DESCRIPTION
**Pull request by jerry-jma**

### CODE REVIEW

File: src/Invitation/InvitationForm.js [hunk 1]
```diff
@@ -1,16 +1,120 @@
...
+            ))}
+          </CardGroup>
+          <Form.Control
+            type="file"
+            ref={hiddenGalleryInput}
+            accept="image/*"
+            style={{ display: 'none' }}
+            onChange={previewGallery}
+            multiple
+          />
```
Comment (oliviaschwerdt) [inline src/Invitation/InvitationForm.js:107]: In your next refactor, you may need to adjust the img attributes in order for the gallery photos to keep their aspect ratio but be clipped to fill the desired area for all screen sizes.

File: src/Invitation/InvitationForm.js [hunk 2]
```diff
@@ -1,16 +1,120 @@
...
+      </Stack>
+      <div>Main Photo</div>
+      <img src={mainPhoto} alt="mainPhoto" width="100%" height="200" />
+      <input
+        type="file"
+        ref={hiddenMainPhotoInput}
+        accept="image/*"
+        style={{ display: 'none' }}
+        onChange={previewMainPhoto}
+      />
```
Comment (oliviaschwerdt) [inline src/Invitation/InvitationForm.js:82]: In your next refactor, you may need to adjust the img attributes in order for the main photo to keep its aspect ratio but be clipped to fill the desired area for all screen sizes.

**Review by oliviaschwerdt — APPROVED** [review]
Hi Jerry! Great job on the Invitation Form! I left a couple comments about image sizing in the Invitation Form, but that should just be a CSS fix, so that may be something to consider in your next refactor.

File: src/Invitation/BrowseTheme.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import BlueButton from '../Common/styled/buttonstyles/BlueButton';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+export default function BrowseTheme() {
+  return (
+    <Container className="mt-5">
+      <Row>
+        <Col xs lg="9">
+          <ProgressBar now={50} />
+          <h2>Wedding Website</h2>
```
Comment (DylanDKim) [inline src/Invitation/BrowseTheme.js:13]: When you get the chance to do more styling, take a look at the common/styled/textstyles folder, all the headers and text you need should be in there, let me know if you need to add one!

**Review by DylanDKim — APPROVED** [review]

**Pull request merged**